gradle/libs.versions.toml

```diff
@@ -5,6 +5,7 @@ reactorCore = "3.4.27-SNAPSHOT"
 asciidoctor = "3.3.2"
 kafka-doc = "28"
 log4j = "2.17.1"
+micrometer = "1.10.3"
```

Comment:
Many thanks for this PR!
micrometer 1.10.4 just got released today.
Would it be possible to use the latest and greatest from this PR by bumping this 1.10.3 to 1.10.4 please?

Reply:
addressed

Reply:
Thanks a lot!